fsleyes_plugin_shimming_toolbox/st_plugin.py

```diff
@@ -1136,7 +1139,7 @@ def create(self):
 
 def create_asterisk_icon(panel):
     bmp = wx.ArtProvider.GetBitmap(wx.ART_INFORMATION)
-    info_icon = os.path.join(__dir_shimmingtoolbox__, 'shimmingtoolbox', 'gui', 'asterisk.png')
+    info_icon = os.path.join(DIR, 'img', 'asterisk.png')
```

Comment:
I'd consider using https://docs.python.org/3/library/importlib.html#importlib.resources.path here. It future-proofs this against different packaging methods.

You use it like this:

```
import shimmingtoolbox.gui.img
[...]
     with importlib.resources.path(shimmingtoolbox.gui.img, 'asterisk.png') as info_icon
```

The `with` is because importlib has to account for the case where code comes in a .zip (python's answer to .jars), but no one ever does that anymore and it's [even discouraged nowadays](https://packaging.python.org/specifications/binary-distribution-format/#is-it-possible-to-import-python-code-directly-from-a-wheel-file), so I think we can just ignore this, and some day they'll come to their senses and we will be able to rewrite this as `info_icon = importlib.resources.path(shimmingtoolbox.gui.img, 'asterisk.png')`